config/jobs/update-drivers-website/update-drivers-website.yaml

```diff
@@ -1,6 +1,6 @@
 periodics:
   - name: update-drivers-website
-    cron: '0 12 * * *'  # every day at 12pm
+    cron: '0 */2 * * *'  # every 2h
```

Comment:
For me is fine. It's a lightweight job. @FedeDP @leogr @zuc what do you think?

Reply:
It's ok for me!